kong/plugins/standard-webhooks/schema.lua

```diff
@@ -0,0 +1,38 @@
+local typedefs = require "kong.db.schema.typedefs"
+
+local PLUGIN_NAME = "standard-webhooks"
+
+local schema = {
+  name = PLUGIN_NAME,
+  fields = {
+    { consumer = typedefs.no_consumer },
+    { protocols = typedefs.protocols_http },
+    {
+      config = {
+        type = "record",
+        fields = {
+          {
+            secret = {
+              type = "string",
+              required = true,
+              description = "Webhook secret",
+              referenceable = true,
+              encrypted = true,
```

Comment:
This does HMAC-SHA256 only right? So shouldn't we check that the secret starts with `whsec_` according to the standard? See table here: https://github.com/standard-webhooks/standard-webhooks/blob/main/spec/standard-webhooks.md#signature-scheme

Also; then maybe the field should be renamed to `secret_v1` to indicate that it is HMAC-SHA256, and leave future updates open to add `secret_v1a` for ed25519.

Reply:
Oh, fair point. Will update this then 🙏